workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-master/css.css

```
@@ -4,10 +4,56 @@
 	font-family: 'Source Sans 3';
 	font-style: normal;
 	font-weight: 400;
-	src: url([$DOCUMENT_URL:/site-initializer/documents/group/fonts/sourceSans3/SourceSans3-Regular.ttf$]);
+	src: url('/documents/d/marketplace/sourcesans3-regular-ttf');
 }
 
 .lfr-layout-structure-item-com-liferay-portal-search-web-category-facet-portlet-categoryfacetportlet,
 .lfr-layout-structure-item-com-liferay-commerce-product-content-search-web-internal-portlet-cpspecificationoptionfacetsportlet {
 	border-radius: 10px;
 }
+
+.lfr-nav-item.nav-item {
+	border-radius: 8px;
+	margin-right: 0.5rem;
+	transition: all 0.3s ease-in-out;
+}
+
+.lfr-nav-item.nav-item .nav-link {
+	border-bottom: transparent solid 3px !important;
+}
+
+.lfr-nav-item.nav-item:hover {
+	background-color: #ffffff !important;
+	font-weight: none !important;
+}
+
+.lfr-nav-item.nav-item.selected.active .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight: 600;
+}
+
+.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
+	border-bottom: #004AD7 solid 3px !important;
+	color: #0053F0 !important;
+	font-weight: 600;
+}
+
+.child-menu.dropdown-menu {
+	border-radius: 8px !important;
+	padding: 0px !important;
+}
+
+.card.main-image {
+	border-radius: 10px !important;
+}
+
+.product-gallery .arrow.next {
+	border-bottom-right-radius: 10px;
+	border-top-right-radius: 10px;
+}
+
+.product-gallery .arrow.prev {
+	border-bottom-left-radius: 10px;
+	border-top-left-radius: 10px;
+}
```

@welitonpe could you please try to sort?

Doesn't need to be perfect, but as much as possible

```suggestion
.card.main-image {
	border-radius: 10px !important;
}

.child-menu.dropdown-menu {
	border-radius: 8px !important;
	padding: 0px !important;
}

.lfr-nav-item.nav-item {
	border-radius: 8px;
	margin-right: 0.5rem;
	transition: all 0.3s ease-in-out;
}

.lfr-nav-item.nav-item .nav-link {
	border-bottom: transparent solid 3px !important;
}

.lfr-nav-item.nav-item:hover {
	background-color: #ffffff !important;
	font-weight: none !important;
}

.lfr-nav-item.nav-item.selected.active .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.lfr-nav-item.nav-item.dropdown:has(.child-menu.dropdown-menu .active.selected) .nav-link {
	border-bottom: #004AD7 solid 3px !important;
	color: #0053F0 !important;
	font-weight: 600;
}

.product-gallery .arrow.next {
	border-bottom-right-radius: 10px;
	border-top-right-radius: 10px;
}

.product-gallery .arrow.prev {
	border-bottom-left-radius: 10px;
	border-top-left-radius: 10px;
}
```